### DISCUSSION TITLE
Diagrama DER

### DISCUSSION THREAD

**Discussion by ZafkielEdt** [FrancoGL/ISPC-FullStack#96] [Show and tell]
@federico42o @pepi1100 @cheaguz @javiercorzo37 @Alejo11Lucero 

Buenas, este es el DER que hice, al final dejo el link por si quieren cambiar o quitar algo.

![DER drawio](https://user-images.githubusercontent.com/66887467/233447801-69ed6962-1703-48cc-94fc-60e4ed8149ec.svg)

https://drive.google.com/file/d/1F6keJwbns7rC0nUpOPWt--8LwXWXCTK0/view?usp=sharing

**Comment by federico42o** [comment]
Para mi esta bastante bien, muy bueno 👍
